shardingsphere-infra/shardingsphere-infra-authority/shardingsphere-infra-authority-common/src/test/java/org/apache/shardingsphere/authority/checker/AuthorityCheckerTest.java

```diff
@@ -0,0 +1,227 @@
+import org.apache.shardingsphere.sql.parser.sql.common.statement.dml.InsertStatement;
+import org.apache.shardingsphere.sql.parser.sql.common.statement.dml.SelectStatement;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Answers;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import javax.sql.DataSource;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.LinkedList;
+import java.util.Map;
+import java.util.Properties;
+import java.util.stream.Collectors;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.notNullValue;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.RETURNS_DEEP_STUBS;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.Silent.class)
+public class AuthorityCheckerTest {
+
+    static {
+        ShardingSphereServiceLoader.register(SQLChecker.class);
+    }
+
+    @Mock(answer = Answers.RETURNS_DEEP_STUBS)
+    private ShardingSphereMetaData metaData;
+
+    @Test
+    public void testCheckSchemaByAllPrivilegesPermitted() {
+        Collection<ShardingSphereUser> users = new LinkedList<>();
+        ShardingSphereUser root = new ShardingSphereUser("root", "", "localhost");
+        users.add(root);
+        AuthorityRuleConfiguration ruleConfig = new AuthorityRuleConfiguration(Collections.emptyList(), new ShardingSphereAlgorithmConfiguration("ALL_PRIVILEGES_PERMITTED", new Properties()));
+        AuthorityRule rule = new AuthorityRule(ruleConfig, Collections.emptyMap(), users);
+        SQLChecker<AuthorityRule> sqlChecker = OrderedSPIRegistry.getRegisteredServices(Collections.singleton(rule), SQLChecker.class).get(rule);
+        assertThat(sqlChecker, notNullValue());
+        // any schema
+        assertThat(sqlChecker.check("db0", new Grantee("root", "localhost"), rule), is(true));
+    }
+
+    @Test
+    public void testCheckSchemaByNative() throws SQLException {
+        Collection<ShardingSphereUser> users = new LinkedList<>();
+        ShardingSphereUser root = new ShardingSphereUser("root", "", "localhost");
+        users.add(root);
+        AuthorityRuleConfiguration ruleConfig = new AuthorityRuleConfiguration(Collections.emptyList(), new ShardingSphereAlgorithmConfiguration("NATIVE", new Properties()));
+        AuthorityRule rule = new AuthorityRule(ruleConfig, createMetaDataMap(users), users);
+        SQLChecker<AuthorityRule> sqlChecker = OrderedSPIRegistry.getRegisteredServices(Collections.singleton(rule), SQLChecker.class).get(rule);
+        assertThat(sqlChecker, notNullValue());
+        // true
+        assertThat(sqlChecker.check("db0", new Grantee("root", "localhost"), rule), is(true));
+        // false
+        assertThat(sqlChecker.check("db1", new Grantee("root", "localhost"), rule), is(false));
+        // false
```

Comment:
I can see these results from assertThat, therefore what's the point of these comments?

Reply:
I will remove it